docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -102,7 +104,24 @@ export const HeroSection = () => {
             </Grid>
           ) : null}
 
-          <TerminalCommand command={frameworkInstallScript} variant="hero" />
+          {platform === 'flutter' ? (
+            <TerminalCommand command={frameworkInstallScript} variant="hero" />
+          ) : (
+            <Tabs>
```

Comment:
Is this something we should be doing with the Hero install script? IMO the tabs are not going to align well with the current UX/UI. We should make sure to at the very least get @dbanksdesign sign off on this change imo

Reply:
I personally don't mind the tabs that much, but I'll leave that up to @dbanksdesign

Reply:
Agree with @reesscot and @calebpollman I don't think we need the tabs on the homepage, it just doesn't feel right.

Reply:
OK. PR's updated!